### DISCUSSION TITLE
add_mesh is slow when called many times on a QtInteractor

### DISCUSSION THREAD

**Discussion by NatanLifshitz** [pyvista/pyvista#6689] [Q&A] [answered]
### First check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the PyVista documentation, with the integrated search.
- [X] I already searched in Google "How to X in PyVista" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I will NOT make duplicate discussions or issues in the future.

### Commit to Help

- [X] I commit to help with one of those options.

### Sample Code What is the problem, question, or error?
Write a short description telling me what you are doing, what you expect to happen, and what is currently happening.

```python
I'm getting significant performance issues when using many meshes with QtInteractor as opposed to Plotter.
Basically, I'm calling add_mesh a few thousand times on QtInteractor at startup to add a few thousand meshes, and it is taking about 30 seconds, as opposed to around 1 second with Plotter. Is there a more performant way to add these meshes to the QtInteractor plotter? Will merging my meshes into less, larger meshes fix this issue or does it not matter?
```

### Description

-

### System Information

```shell
pyvista~=0.44.1
PySide6~=6.7.3
pyvistaqt~=0.11.1
```

**Comment by NatanLifshitz** [comment, marked as answer]
See [Discussion on Slack](https://app.slack.com/client/TBA0M65B2):

> Bane
  [Monday at 4:52 PM](https://pyvista.slack.com/archives/CBA5095UH/p1727704365962079?thread_ts=1727693045.707899&cid=CBA5095UH)
I recommend using composite rendering with that many meshes regardless (MultiBlock). I’m not familiar with those performance differences but I wonder if delaying rendering helps (specifying render=false in the add_mesh call)

**Reply by NatanLifshitz** [reply]
Using `Multiblock` and `render=false` solved the issue
